package.json

```diff
@@ -51,9 +52,12 @@
     "@types/react-test-renderer": "^16.0.3",
     "@types/styled-components": "^4.0.3",
     "babel-core": "7.0.0-bridge.0",
+    "buffer-crc32": "^0.2.13",
     "codecov": "^3.1.0",
     "cross-spawn": "^6.0.5",
     "glob": "^7.1.3",
+    "jest-puppeteer": "^3.5.1",
+    "node-jq": "^1.3.1",
```

Comment:
this isn't used

Reply:
oh cool, then the `buffer-crc32` is not needed